index.d.ts

```diff
@@ -0,0 +1,92 @@
+export class Client {
+  public config: Configuration;
```

Comment:
Unlike for example C#, all properties are public by default in TypeScript, so it's not necessary to mark everything as public. Not a usual practice since it makes the code more verbose than it needs to be.